src/DefaultBuilder/src/WebHostEnvironment.cs

```diff
@@ -106,13 +105,35 @@ public void ResolveFileProviders(IConfiguration configuration)
             }
         }
 
+        private string ResolveWebRootPath(string webRootPath, string contentRootPath)
+        {
+            if (string.IsNullOrEmpty(webRootPath))
+            {
+                return contentRootPath;
```

Comment:
Shouldn't this be Path.GetFullPath(contentRootPath)?

Reply:
Yep -- good call!